workspaces/cogito-react-ui/source/__mocks__/qrcode.react.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const MockedQRCode = ({ value }) => <div>{ value }</div>
+
+export default MockedQRCode
```

Comment:
As a general rule, we avoid using default exports. See: https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad for some reasoning-behind.

Reply:
Agreed about the general rule, but in this case @Hacklor is mocking the qrcode.react library, which just happens to use default exports.